_data/accessibility-tests/collection.yml

```diff
@@ -0,0 +1,85 @@
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.2
+    wcag_criterion: 2.4.6
+# Keyboard navigation tests
+  - summary: Collection content is accessible using only keyboard. 
+    summary_additional: When you use a keyboard, you can navigate to links, buttons and content, and navigate away.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.2
+    wcag_criterion: 2.1.1
+  - summary: Collection content does not trap keyboard focus.  
+    summary_additional: When you tab through collection content with a keyboard, you can easily move in and out of collection elements to get to other content.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.2
+    wcag_criterion: 2.1.2
+  - summary: Focus follows logical content order. 
+    summary_additional: When you tab through a collection with a keyboard, focus on links and other interactive elements stays in logical order in relation to the other content.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.2
+    wcag_criterion: 2.4.3
+  - summary: Focus indicator is clearly visible. 
+    summary_additional: When you use a keyboard to focus on a collection, you will see a visible outline or other clear indication of focus for interactive elements.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.2
+    wcag_criterion: 2.4.7
+# Screen reader tests
+  - summary: Screen reader only announces icons or graphics marked with alt text.
+    summary_additional: When you open the collection with screen reader, you don't hear any information about icons or images marked as decorative. 
+    test_status: conditional
+    test_type: screen_reader
+    version_tested: 3.8.2
+    wcag_criterion: 1.1.1
+  - summary: Screen reader announces collection region. 
+    summary_additional: When using a screen reader to access a collection, you hear the hierarchy of headings, lists and links as they are presented on the page. You can also use shortcut keys in the screen reader to navigate the items in the collection. 
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.2
+    wcag_criterion: 1.3.1
+  - summary: Collection content is read in the same order that it appears on the page. 
+    summary_additional: When you use a screen reader and navigate through a collection,  you hear each piece of content in the same order in which it appears on the page (e.g., left to right and top to bottom for English pages).
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.2
+    wcag_criterion: 1.3.2
+  - summary: Screen reader provides direct description of links. 
+    summary_additional: When you navigate to a link in a collection with a screen reader, you can understand where the link will take you. 
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.2
+    wcag_criterion: 2.4.4
```

Comment:
**Todo:** We decided to update this test to `conditional` since users are responsible for adding the links.